frontends/web/src/components/dialog/dialog.module.css

```diff
@@ -236,11 +228,40 @@
     width: 100%;
 }
 
+@media (min-width: 769px) {
+    .closingModal {
+        background: red;
+    }
+
+    .closingOverlay {
+        display: none;
+    }
```

Comment:
Without this additional class, a user on desktop will be unable to scroll the app immediately for 240ms after they closed the modal. (they need to wait for 240ms upon closing, which isn't ideal).